### DISCUSSION TITLE
BTC Price from the command line

### DISCUSSION THREAD

**Discussion by tcharding** [rust-bitcoin/rust-bitcoin#3690] [Other Bitcoin-related crates]
During this bull run you might find yourself opening some web page to check the price (too often). Here is a quick and dirty tool that grabs the price of the last trade from Kraken.

https://github.com/tcharding/btc-price

Written one Saturday morning with ChatGPT.

**Comment by storopoli** [comment]
`curl rate.sx/btc@10d`.
Also works with other cryptocurrencies.

**Comment by yancyribbens** [comment]
I like to follow https://insights.braiins.com/en from time to time which shows the network stats and also a price correlation.  The braiins guys did a nice job with this tool imo.
